src/order/rel_iso/group.lean

```diff
@@ -37,4 +37,12 @@ lemma mul_apply (e₁ e₂ : r ≃r r) (x : α) : (e₁ * e₂) x = e₁ (e₂ x
 
 @[simp] lemma apply_inv_self (e : r ≃r r) (x) : e (e⁻¹ x) = x := e.apply_symm_apply x
 
+/-- The tautological action by `r ≃r r` on `α`. -/
+instance apply_mul_action : mul_action (r ≃r r) α :=
+{ smul := coe_fn,
+  one_smul := λ _, rfl,
+  mul_smul := λ _ _ _, rfl }
+
+@[simp] lemma smul_def (f : r ≃r r) (a : α) : f • a = f a := rfl
```

Comment:
Can you add the `has_faithful_smul` instance that normally comes with these instances?